.github/workflows/enforce-dependency-review.yml

```
@@ -0,0 +1,39 @@
+name: Enforce Dependency Review
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

Same reasoning as health check PR